articles/logic-apps/quickstart-create-example-consumption-workflow.md

```diff
@@ -138,6 +138,11 @@ This example uses an RSS trigger that checks an RSS feed, based on the specified
 
    This step instantly publishes your logic app resource and workflow live in the Azure portal. However, the trigger only checks the RSS feed without taking any other actions. So, you need to add an action to specify what you want to happen when the trigger fires.
 
+1. On the logic app code view,  
```

Comment:
@LimitlessDevv: Thanks for your contribution! I'd move this as a step before the last step (save the workflow).

```suggestion
1. On the designer toolbar, select **Code view**. In the code editor, change the line **`"feedUrl": "@{encodeURIComponent(encodeURIComponent(`https://feeds.a.dj.com/rss/RSSMarketsMain.xml'))}" to the following version:

   **`"feedUrl": "@{encodeURIComponent('https://feeds.a.dj.com/rss/RSSMarketsMain.xml')}"

   This change is necessary to remove double-encoding behavior, which requires manual correction.

```